report/src/main/webapp/assets/sass/_variables.scss

```diff
@@ -60,6 +60,7 @@ $warning: #f0ad4e;
 $rebased: #0097fe;
 $failed: #bb5a5a;
 $passed: #6f9f00;
+$conditionallyPassed: #aacc2d;
```

Comment:
I'm not sure about marking conditionally passed as a separate statistic and color. 
There should be proper filter to show conditionally passed, but let's not introduce additional color for now.